pages/advanced/smart-account-bug-bounty.md

```diff
@@ -25,8 +25,9 @@ The scope of the bug bounty program includes the core contracts related to the f
 * _v1.1.1_ ([Release details](https://github.com/safe-global/safe-contracts/releases/tag/v1.1.1), [README](https://github.com/safe-global/safe-contracts/blob/v1.1.1/README.md))
 
 The scope of the bug bounty also includes officially supported Safe Modules:
-* _v1.0.0_ [Allowance Module](https://github.com/safe-global/safe-modules/tree/47e2b486b0b31d97bab7648a3f76de9038c6e67b/allowances)
+* _v0.1.1_ [Allowance Module](https://github.com/safe-global/safe-modules/tree/allowance/v0.1.1/modules/allowances)
```

Comment:
Should it include 0.1.0 as well?

Reply:
Good question - I guess it should since it is actively used in the wild.